domains/arty.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "desariah-airbourne",
+           "email": "anthony@folley.io",
+           "discord": "1006588164003086497"
+        },
+    
+        "record": {
+            "CNAME": "desariah-airbourne.github.io"
```

Comment:
404 error page, please give a preview of your website

Reply:
Added GitHub repo 👍